tests/aws-cpp-sdk-transfer-unit-tests/TransferUnitTests.cpp

```diff
@@ -1,13 +1,23 @@
+#ifdef _WIN32
```

Comment:
why do we need this? feels like this test shoudlnt need to know if its on windows or not